lighthouse-core/config/config.js

```diff
@@ -162,6 +161,34 @@ function assertValidGatherer(gathererInstance, gathererName) {
   }
 }
 
+
+/**
+ * Validate the LH.Flags
+ * @param {LH.Flags} flags
+ */
+function assertValidFlags(flags) {
+  // @ts-expect-error Checking for removed flags
+  if (flags.emulatedFormFactor || flags.internalDisableDeviceScreenEmulation) {
```

Comment:
I'd expect this to break `yarn test-devtools`. We have a circular dep here. must change CDT frontend code to pass our `yarn test-devtools`...but can't do that until we roll v7 ...but cant do that until this PR lands....

or we hack around in devtools entry, to change the flags passed in.